internal/testutil/acc/pre_check.go

```diff
@@ -45,10 +46,26 @@ func PreCheckAtlasUsername(tb testing.TB) {
 	}
 }
 
-func PreCheckTeamsIds(tb testing.TB) {
-	if os.Getenv("MONGODB_ATLAS_TEAMS_IDS") == "" {
-		tb.Skip("`MONGODB_ATLAS_TEAMS_IDS` must be set for Projects acceptance testing")
+func PreCheckProjectTeamsIds(tb testing.TB, min int) {
```

Comment:
```suggestion
func PreCheckProjectTeamsIds(tb testing.TB, minProjectCount int) {
```

Reply:
I can change it, although I tried to follow "go idiomatic" in the sense the I think "min" in this context is clear, but I agree I can make it more specific here.